<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SfcWin</class>
 <widget class="QWidget" name="SfcWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SFC窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>站別名稱</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>返回结果</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>工单号码</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>测试Web Service</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,1">
      <item row="6" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>小板号码</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>小板号码数量</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>FinishPrintInfo</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>GetTime</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>板边号码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>绑定资料</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
